src/init/starship.fish

```diff
@@ -18,4 +18,4 @@ function fish_mode_prompt; end
 export STARSHIP_SHELL="fish"
 
 # Set up the session key that will be used to store logs
-export STARSHIP_SESSION_KEY=(::STARSHIP:: session)
+export STARSHIP_SESSION_KEY=(string sub -s1 -l16 (random)(random)(random)(random)(random)0000000000000000)
```

Comment:
```suggestion
export STARSHIP_SESSION_KEY=(random 1000 9999)(random 1000 9999)(random 1000 9999)(random 1000 9999)'
```
How about this to get rid of the need to cut the string?

Reply:
```suggestion
export STARSHIP_SESSION_KEY=(random 10000000000000 9999999999999999)
```

Let's go one step further. Since it's not meant to be secure, use a 16-digit number, and we're done.
I first copy/paste-and-adapt the code from Bash, but fish can do better, so let's do it. And it's lot easier to read.

Reply:
Is that going to work on <64 Bit systems? 

New benchmarks:
| Command | Mean [ms] | Min [ms] | Max [ms] | Relative |
|:---|---:|---:|---:|---:|
| `fish -c 'export X=(starship session)'` | 17.4 ± 9.2 | 5.6 | 33.9 | 1.00 |
| `fish -c 'export X=(random 10000000000000 9999999999999999)'` | 18.4 ± 6.9 | 3.9 | 30.1 | 1.06 ± 0.68 |

| Command | Mean [ms] | Min [ms] | Max [ms] | Relative |
|:---|---:|---:|---:|---:|
| `fish -c 'export X=(starship session)'` | 6.5 ± 0.4 | 5.9 | 9.4 | 1.35 ± 0.15 |
| `fish -c 'export X=(random 10000000000000 9999999999999999)'` | 4.8 ± 0.4 | 4.1 | 5.9 | 1.00 |

Reply:
Yes it should. Fish [parses arguments as `long long`](https://github.com/fish-shell/fish-shell/blob/master/src/builtin_random.cpp#L63), and standard C++ says that it is [at least 64 bits](https://en.cppreference.com/w/cpp/language/types#Properties)